src/code/z_play.c

```diff
@@ -168,7 +168,11 @@ void Play_SetupTransition(PlayState* this, s32 transitionType) {
                 break;
 
             default:
+#if OOT_NTSC
+                HUNGUP_AND_CRASH("../z_play.c", 2287);
+#else
                 HUNGUP_AND_CRASH("../z_play.c", 2290);
+#endif
```

Comment:
line numbers could start to get unruly, I agree. we may start including multiple args in macros that need a line number for different versions, idk

Reply:
maybe something like 2287 + LINE_NUMBER_SHIFT where LINE_NUMBER_SHIFT is defined with a bunch of ifdefs instead? Not really sure, I'd like to wait for more examples before deciding how to handle this